### DISCUSSION TITLE
求助大佬们，使用lucky大吉无法外网访问alist

### DISCUSSION THREAD

**Discussion by FYboss** [AlistGo/alist#6602] [Q&A]
![VeryCapture_20240609221103](https://github.com/alist-org/alist/assets/33796818/4cc37114-77ee-4481-b04c-5210ddc51257)
![image](https://github.com/alist-org/alist/assets/33796818/0ee9cbf4-0cf2-49cd-ac7c-6821a0fa750f)
![image](https://github.com/alist-org/alist/assets/33796818/da3fad67-b780-4206-9ff1-ba8d03ae4b74)

如图显示，内网访问是正常的，但是使用域名访问就不行，同机器的其他docker容器外网访问也正常，只有alist不行，试了很多方法也不行，也没找到和我类似的问题，不清楚是哪里的问题导致，config配置文件没有修改过，是默认的，

**Comment by ppllpig** [comment]
关注，搜到一个npm的解决办法，实测lucky用不了
https://www.huluohu.com/posts/927/
